docs/_notebooks/Resnet50-example.html

```diff
@@ -1920,4 +1920,4 @@ <h3 id="What's-next">
    app.initialize({version: "1.0.4", url: {base: ".."}})
   </script>
  </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
These doc changes should be removed

Reply:
I am not sure how is this being introduced. I already removed any new lines at the end but it does not shows up in git diff.

Reply:
they are probably from an old commit. Not sure of the best way to remove the changes, maybe rebase?

Reply:
fixed it